test/fixtures/hello-cloud/stateful_set.yml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: v1
+kind: Service
```

Comment:
What's the use of having a service in this fixture?

Reply:
Good point. I'll remove it